src/services/repair-request.service.ts

```diff
@@ -0,0 +1,25 @@
+import prisma from "@/lib/prisma";
+import { RepairRequest } from "@/types";
+
+const insert = async (insertDetails: RepairRequest & { createdBy: string }) => {
+  const { images, ...rest } = insertDetails;
+  const repairRequest = await prisma.repairRequest.create({
+    data: {
+      ...rest,
+      images: {
+        create: images?.map((image) => {
+          return {
+            s3Key: image
+          };
+        })
+      }
+    }
+  });
+  return repairRequest;
+};
+
+const repairRequestService = {
```

Comment:
I think a better testable pattern here would to be define a class that implements an interface containing the method `insert`